### ISSUE TITLE
Wrong font weight on table headers

### ISSUE THREAD

**Issue by gabbsmo** [n8design/htwoo#125]
**Describe the bug**
`<th>` does not have a defined `font-weight` in htwoo-core. Compare [Pattern Lab](https://lab.n8d.studio/htwoo/htwoo-core/?p=atoms-table) (700, in Edge's user agent style) with [Fluent React](https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist) (600).

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Edge
 - Version: 122

**Comment by gabbsmo** [comment]
My workaround: 

```
.hoo-table.compact tfoot th, .hoo-table.compact tbody th, .hoo-table.compact thead th {
    font-weight: 600;
}
```

**Comment by StfBauer** [comment]
Thank for finding this issue.

**Comment by StfBauer** [comment]
Fixed in in 2.4. 0
